src/Raven.Server/Utils/BackupUtils.cs

```diff
@@ -77,6 +81,13 @@ internal static BackupInfo GetBackupInfo(BackupInfoParameters parameters)
         };
     }
 
+    internal static PeriodicBackupStatus GetBackupStatusFromCluster(ServerStore serverStore, string databaseName, long taskId)
```

Comment:
private?

Reply:
I have completely removed the overload of this method.